npm-shrinkwrap.json

```diff
@@ -135,7 +135,7 @@
     },
     "@sinonjs/formatio": {
       "version": "2.0.0",
-      "resolved": "https://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz",
```

Comment:
@vbudhram need to keep these as `https`

Reply:
No idea why that was the only one that didn't get updated 🤷🏽‍♂️